subsys/suit/mci/src/suit_mci_nrf9280.c

```diff
@@ -0,0 +1,716 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
```

Comment:
In this case - 2024 will be better :slightly_smiling_face:

Reply:
Fixed.